auth-web/src/components/auth/common/NumberedCompanyTooltip.vue

```diff
@@ -1,11 +1,13 @@
 <template>
   <v-tooltip top max-width="450px" light content-class="tooltip">
     <template v-slot:activator="{ on }">
-      <span v-on="on" class="tooltip-text"><u>numbered Benefit Company</u></span>
+      <span v-on="on" class="tooltip-text">numbered Benefit Company</span>
```

Comment:
numbered -> Numbered?

Reply:
Designs actually show this tooltip lowercase on the `numbered`, perhaps because it's slotted into a sentence. Will follow up on this!